lib/sycamore/sycamore/reader.py

```diff
@@ -280,11 +282,11 @@ def opensearch(
         client_params = OpenSearchReaderClientParams(os_client_args=os_client_args)
         query_params = (
             OpenSearchReaderQueryParams(
-                index_name=index_name, query=query, reconstruct_document=reconstruct_document, kwargs=kwargs
+                index_name=index_name, query=query, reconstruct_document=reconstruct_document, kwargs=query_kwargs
```

Comment:
Can you add a comment here as to why we've switched this? It is a bit inscrutable.

Reply:
Discussed offline, I've added a docstring about query_kwargs above already